### DISCUSSION TITLE
How to preview the QuantEcon theme?

### DISCUSSION THREAD

**Discussion by choldgraf** [executablebooks/meta#264] [Q&A]
@mmcky I was just chatting with @hamelsmu , who is trying out Jupyter Book and was interested to see whether it's possible to use other themes than the default book theme. I know that you have done a lot of work on the QuantEcon side with your theme, and I pointed him to it, but wasn't sure were you could preview a live version of the theme.

Do you know:

1. Where there's a live version of the QE theme someone could look at?
2. A place w/ instructions for how somebody could use that theme with Jupyter Book? (this makes me wonder if we should write up these instructions as a `discussion` here and refer to it as informal documentation until we figure out the long term theming story)

**Comment by hamelsmu** [comment]
I think I figured it out.  I'll open a pull request.

**Reply by hamelsmu** [reply]
https://github.com/QuantEcon/quantecon-book-theme/pull/88

**Comment by hamelsmu** [comment]
I just need to figure out how to get rid of this thing 
![image](https://user-images.githubusercontent.com/1483922/109106732-3d1f9380-76e5-11eb-8f4e-24ef42253b5c.png)

**Reply by hamelsmu** [reply]
Ehh it seems like that particular icon is embedded deep into the theme like in the CSS file :(  If anyone has any tips on how to remove that that would be useful.  I guess I could fork the project and try to remove that element from everywhere it appears

**Reply by choldgraf** [reply]
@hamelsmu as a stop-gap could you try adding custom CSS to select that image w/ CSS and add a `display: none;` rule?

**Reply by mmcky** [reply]
@hamelsmu @choldgraf -- the `quantecon-book-theme` was really our `learning` theme project so it has a lot of fixed assets at the moment :-(. One of the jobs is to disentangle some of those elements and make them `parameterized` if others would like to make use of that theme which is why we have kept it in `QuantEcon` organisation for now. 

@DrDrij is also working on a couple of new themes based off of `sphinx-book-theme` and trying to build them based on an `inheriteance` type model.

**Comment by mmcky** [comment]
@hamelsmu here is a live project making use of the theme. Others will be coming online soon.

https://python-programming.quantecon.org/intro.html
